src/components/clusterWizard/wizardTransition.ts

```diff
@@ -14,7 +14,10 @@ export type ClusterWizardStepsType =
   | 'networking'
   | 'review';
 
-export const CLUSTER_WIZARD_FIRST_STEP: ClusterWizardStepsType = 'baremetal-discovery';
+export const getClusterWizardFirstStep = (): ClusterWizardStepsType => {
+  const searchParams = new URLSearchParams(window.location.search);
```

Comment:
Another approach can be via react-router, either by `useLocation()` or by parsing query params at the top of component hierarchy (close to Router). The so far proposed solution requires minimum code changes.